platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/merkle/VirtualRootNode.java

```diff
@@ -1317,7 +1396,8 @@ public void warmInternalNodesForLeaf(final long leafPath) {
             siblingPath = getSiblingPath(path);
 
             if (siblingPath != INVALID_PATH) {
-                records.warmInternalRecord(siblingPath);
+                // Pre-load sibling node hash
+                records.findHash(siblingPath);
```

Comment:
This needs to be checked with Oleg. Warmups do have positive effect on performance, so they can't be removed altogether. The difference between warmInternalRecord() and findHash() is the former doesn't deserialize bytes to hashes. How much is saved on skipping it is an open question.